packages/react/src/components/TextArea/TextArea.js

```diff
@@ -183,6 +229,11 @@ TextArea.propTypes = {
    */
   cols: PropTypes.number,
 
+  /**
+   * Specify the method used for calculating the counter number
+   */
+  counterMode: PropTypes.oneOf(['default', 'words']),
```

Comment:
Instead of default and words, would it make more sense to be a little more specific and use `character` and `word`